extensions/blocks/membership-button/membership-button.php

```diff
@@ -0,0 +1,17 @@
+<?php // phpcs:disable Squiz.Commenting.FileComment.Missing
+/**
+ * Memberships block.
+ *
+ * @since 7.3.0
+ *
+ * @package Jetpack
+ */
+
+require_once JETPACK__PLUGIN_DIR . '/modules/memberships/class-jetpack-memberships.php';
+
+jetpack_register_block(
```

Comment:
I think the generic `is_active` before _any_ blocks are loaded should protect from this?

https://github.com/Automattic/jetpack/blob/e54806a976d63db1c66f295187b2f507ba2e449e/class.jetpack-gutenberg.php#L401

Reply:
No, because in this case we don't want the block to load when we are in development mode (but for some of our other blocks, like Business blocks, we do).

Reply:
Done!